### PR TITLE
Add possibility to ignore person message

### PR DESCRIPTION
**Pull request by anethus**
That is acc wild - all chars from speific acc will share that list

User can filter message from Ignored Person

### CODE REVIEW

**Comment by anethus** [comment]
ignoreList2

File: src/Game/Scenes/GameSceneInputHandler.cs [hunk 1]
```diff
@@ -602,6 +602,13 @@ private bool OnLeftMouseUp()
                             CommandManager.OnHueTarget(selectedEntity);
                         }
 
+                        break;
+                    case CursorTarget.IgnorePlayerTarget:
```
Comment (anethus) [inline src/Game/Scenes/GameSceneInputHandler.cs:606]: For targeting Mobile

File: src/Game/Managers/TargetManager.cs [hunk 2]
```diff
@@ -391,6 +392,13 @@ public static void Target(uint serial)
 
                         ClearTargetingWithoutTargetCancelPacket();
 
+                        return;
+                    case CursorTarget.IgnorePlayerTarget:
```
Comment (anethus) [inline src/Game/Managers/TargetManager.cs:396]: For targeting HealthBar

**Comment by jaedan** [comment]
I rebased this and opened a new PR - can you verify this looks right to you? https://github.com/UOOutlands/ClassicUO/pull/23

**Pull request closed without merging**